Makefile

```diff
@@ -85,7 +85,7 @@ ALL_COPY := $(subst $(HERE),$(VENV),$(SRC_FILES) $(DOC_FILES) $(RCP_FILES) $(ROO
 $(foreach s,$(ALL_COPY),$(eval $s: $(VENV)/bin/$(PYTHON_EXE) $(subst $(VENV),$(HERE),$s)))
 
 $(ALL_COPY):
-	@test -d "$(dir $@)" || mkdir -p $(dir $@)
+	@mkdir -p $(dir $@)
```

Comment:
JSYK there was more than one instance of `test -d || mkdir -p` in the Makefile, but I didn't tag them all.